gallery/src/styles/NxPageTitle/NxPageTitleSubtitleExample.html

```diff
@@ -0,0 +1,24 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+<div class="nx-page-title">
+  <h1 class="nx-h1">
+    Page Title
+  </h1>
+  <h2 class="nx-h2 nx-page-title__sub-title">
```

Comment:
Technically I feel like this and the `h1` should be wrapped together in an `hgroup`.  Otherwise everything from here down to the next header will be considered a section under the `h2`, outline wise.

Reply:
> Technically I feel like this and the `h1` should be wrapped together in an `hgroup`. Otherwise everything from here down to the next header will be considered a section under the `h2`, outline wise.

Good point: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/78ca66b4343fccc6421e96044299e5583a7f2486